lib/jekyll/filters.rb

```diff
@@ -223,7 +224,9 @@ def group_by(input, property)
     def where(input, property, value)
       return input unless input.is_a?(Enumerable)
       input = input.values if input.is_a?(Hash)
-      input.select { |object| Array(item_property(object, property)).map(&:to_s).include?(value.to_s) }
+      input.select do |object|
+        Array(item_property(object, property)).map(&:to_s).include?(value.to_s)
```

Comment:
Sorry for the late reply @ayastreb, I didn't get an email for some reason and just randomly checked to see if this had feedback.  That said, I guess keep it as `Array`, I didn't think they were doing a cheap flatten, but it seems to me now that they are.  I don't quite agree since ([blah].flatten(1) is more explicit to a non-programmer but I digress.

Lets get some context on this from @parkr or anybody else on @jekyll/core who might have more info.